internal/storage/db/internal/db.go

```diff
@@ -570,6 +568,31 @@ func (s *dbStorage) Disable(ctx context.Context, policyKey ...string) (uint32, e
 	return uint32(affected), nil
 }
 
+func (s *dbStorage) Enable(ctx context.Context, policyKey ...string) (uint32, error) {
+	mIDs := make([]namer.ModuleID, len(policyKey))
+	events := make([]storage.Event, len(policyKey))
+	for idx, pk := range policyKey {
+		mIDs[idx] = namer.GenModuleIDFromFQN(namer.FQNFromPolicyKey(pk))
+		events[idx] = storage.Event{Kind: storage.EventAddOrUpdatePolicy, PolicyID: namer.GenModuleIDFromFQN(namer.FQNFromPolicyKey(pk))}
+	}
+
+	res, err := s.db.Update(PolicyTbl).Prepared(true).
```

Comment:
Can we be sure that the `disabled` field of the policy definition is not `true`? It's possible that someone uses the old `AddOrUpdatePolicy` RPC to disable a policy and then later uses this RPC to enable it. In that case, the policy definition would still remain disabled as far as the Cerbos compiler is concerned.